ui/pages/routes/routes.component.js

```diff
@@ -98,6 +99,9 @@ const OnboardingFlow = mmLazy(() =>
 const Lock = mmLazy(() => import('../lock'));
 const UnlockPage = mmLazy(() => import('../unlock-page'));
 const RestoreVaultPage = mmLazy(() => import('../keychains/restore-vault'));
+const ImportSrpPage = mmLazy(() =>
+  import('../../components/multichain/multi-srp/import-srp/import-srp'),
```

Comment:
We should probably follow the same pattern than for other pages here and move it in `ui/pages/` folder?